src/github.com/getlantern/peerscanner/web.go

```diff
@@ -149,12 +149,29 @@ func clientIpFor(req *http.Request, name string) string {
 	ip := strings.TrimSpace(ips[0])
 	// TODO: need a more robust way to determine when a non-fallback host looks
 	// like a fallback.
-	if !isFallback(name) && strings.HasPrefix(ip, "128.199") {
+	hasFallbackIp := isFallbackIp(ip)
+	if !isFallback(name) && hasFallbackIp {
 		log.Errorf("Found fallback ip %v for non-fallback host %v", ip, name)
 		return ""
-	} else if isFallback(name) && !strings.HasPrefix(ip, "128.199") {
+	} else if isFallback(name) && !hasFallbackIp {
 		log.Errorf("Found non-fallback ip %v for fallback host %v", ip, name)
 		return ""
 	}
 	return ip
 }
+
+func isFallbackIp(ip string) bool {
+	for _, prefix := range fallbackIPPrefixes {
+		if strings.HasPrefix(ip, prefix) {
+			return true
+		}
+	}
+
+	return false
+}
+
+var fallbackIPPrefixes = []string{
+	"128.199",
+	"178.62",
+	"188.166",
+}
```

Comment:
Yeah, I don't think IP matching can work for no.2.

Also, it's a maintenance issue.  When adding fallbacks (esp. if they're in different datacenters) we must check whether their IPs are included in the IP ranges hardcoded into peerscanner.  We can only do this after they've launched.  This makes adding flashlight servers a multi-step process.

Then again, I wouldn't mess much further with this until (i) we find this hurts much, or (ii) we've decided on peers.  For example, if we stop supporting give mode peers then we won't need to tell peers from fallbacks at all.